app.js

```diff
@@ -2,7 +2,7 @@ import { app, errorHandler } from "mu";
 import bodyParser from "body-parser";
 import { Delta } from "./lib/delta";
 import { STATUS_SUCCESS, STATUS_FAILED, STATUS_SCHEDULED } from "./constants";
-import { loadTask, createTask, isTask } from "./lib/task";
+import { loadTask, createTask, isTask, isTaskExist } from "./lib/task";
```

Comment:
nit picking, but `taskExists` or `doesTaskExist` would be a better name

Reply:
renamed